docusaurus/docs/identities/enrolling.md

```diff
@@ -57,15 +57,15 @@ confident you understand the risks involved in doing so.
 
 ### 3rd Party CA - One Time Token
 
-This process is similar to the One Time Token flow from above. This flow expects that a private key and certificate have
+The flow allows you to pre-create identities for a 3rd party CA with distinct role attributes. This flow is similar to the One Time Token flow from above except that it expects that a private key and certificate have

```

Comment:
> with distinct role attributes

these are extra words that aren't really helping and are just extra. i think it's more clear to just leave these words out

Reply:
I'm trying to articulate a key difference between CA auto and CA OTT methods:
* CA Auto: all auto-created identities have the same role attributes
*  CA OTT: all pre-created identities have distinct role attributes